Src/BootCamp.Chapter/Player.cs

```diff
@@ -17,25 +18,27 @@ public class Player
         /// </summary>
         private const int baseCarryWeight = 30;
 
-        private string _name;
-        private int _hp;
+        public string Name { get; }
+        public int Hp { get; private set; }
```

Comment:
Changed it all to explicit, Kai mentioned that if it was implicit it would be set in the ctor and I might have gotten the wrong impression that it would only be then.